schemas/edges/is_produced_by.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": ["_from", "_to"],
+  "description": "An entity produced from something else.",
+  "additionalProperties": true,
+  "properties": {
+    "_from": {
+      "type": "string",
+      "description": "The ID of a vertex that produces something else."
```

Comment:
I know we want to keep this generic but it would be helpful to have at least one example of the relationship (even if it's incomplete). IIRC reactions -> is_produced_by -> gene_complexes?

Reply:
Added example